oonib.conf.example

```diff
@@ -30,12 +30,13 @@ main:
     tor_datadir: null
 
     bouncer_endpoints:
-    - {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
+    #- {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
     - {type: tcp, port: 10080}
-    - {type: onion, hsdir: bouncer}
+    - {type: onion, hsdir: /tmp/bouncer}
```

Comment:
Question: so, I understand that `/tmp/bouncer` should fix the error that I reported, right? Why was the `tls` entry commented out? If I recall correctly, when firstly testing, I received an exception because certificate was not there and then I did generate the certificate using the correct script, and then everything worked correctly AFAICT. If possible, thus, I'd leave the `tls` line not commented, such that by default one tests as much features as possible of oonib.